Iceberg-TipUI/IceTipHiedraHistoryBrowser.class.st

```diff
@@ -0,0 +1,28 @@
+"
+I add a Hiedra column to the browser defined by my superclass. This column visualizes the relationship between commits in the history list.
+"
+Class {
+	#name : #IceTipHiedraHistoryBrowser,
+	#superclass : #IceTipHistoryBrowser,
+	#category : #'Iceberg-TipUI-View'
+}
+
+{ #category : #initialization }
+IceTipHiedraHistoryBrowser >> initializeCommitList [
+
+	"Add the Hiedra column at the beginning. The Datasource has the logic to render it."
+	commitList widget addColumn:
+		(IceTipTableColumn new
+			id: ' ';
+			yourself).
```

Comment:
@tinchodias Would it be enough to make this line to execute only for Pharo8?

Reply:
The other option is better